README.md

```diff
@@ -7,56 +7,80 @@
 
-> This project is in a beta state and API might still change a bit
+## What Is ZITADEL
 
-## What Is It
+**ZITADEL** is a "Cloud Native Identity and Access Management'' solution built for the cloud era. ZITADEL uses a modern software stack consisting of [**Golang**](https://golang.org/), [**Angular**](https://angular.io/) and  [**CockroachDB**](https://www.cockroachlabs.com/) as sole storage and follows an event sourced pattern.
 
-**ZITADEL** is a "Cloud Native Identity and Access Management" solution. All server side components are written in [**Go**](https://golang.org/) and the management interface, called **Console**, is written in [**Angular**](https://angular.io/).
-
-We optimized **ZITADEL** for the usage as "service provider" IAM. By "service provider" we think of companies who build services for e.g SaaS cases. Often these companies would like to use an IAM where they can register their application and grant other people or companies the right to self manage a set of roles within that application.
+We built **ZITADEL** not only with the vision of  becoming a great open source project but also as a superb platform to support developers building their applications, without need to handle secure user login and account management themselves.
 
 ## How Does It Work
 
 We built **ZITADEL** around the idea that the IAM should be easy to deploy and scale. That's why we tried to reduce external systems as much as possible.
 For example, **ZITADEL** is event sourced but it does not rely on a pub/sub system to function. Instead we built all the functionality right into one binary.
 **ZITADEL** only needs [**Kubernetes**](https://kubernetes.io/) for orchestration and [**CockroachDB**](https://www.cockroachlabs.com/) as storage.
 
-## Why Another IAM
-
-In the past we already built a closed sourced IAM and tested multiple others. With most of them we had some issues, either technology, feature, pricing or transparency related in nature. For example we find the idea that security related features like **MFA** should not be hidden behind a paywall or a feature price.
-One feature that we often missed, was a solid **audit trail** of all IAM resources. Most systems we saw so far either rely on simple log files or use a short retention for this.
+## Features of ZITADEL platform
+
+* Authentication
+    * OpenID Connect 1.0 Protocol (OP),  OAuth 2.0 Protocol (OP)
+    * Username / Password
+    * Machine-to-machine (JWT profile)
+    * Passwordless/WebAuthN (FIDO2)
+* Multifactor authentication with OTP, U2F
+* Federation with OpenID Connect 1.0 Protocol (RP), OAuth 2.0 Protocol (RP), Google
+* Authorization via Role Based Access Control (RBAC)
+* Identity Brokering
+* Delegation of roles to other organizations for self-management
+* Strong audit trail of all IAM resources
+* User interface for administration
+* APIs for Management, Administration, and Authentication
+* Policy configuration and enforcement
```

Comment:
```suggestion
## Features of ZITADEL platform

* Authentication
    * OpenID Connect 1.0 Protocol (OP)
    * Username / Password
    * Machine-to-machine (JWT profile)
    * Passwordless with FIDO2
* Multifactor authentication with OTP, U2F
* Federation with OpenID Connect 1.0 Protocol (RP), OAuth 2.0 Protocol (RP)
* Authorization via Role Based Access Control (RBAC)
* Identity Brokering
* Delegation of roles to other organizations for self-management
* Strong audit trail for all IAM resources
* User interface for administration
* APIs for Management, Administration, and Authentication
* Policy configuration and enforcement
```

Reply:
- Should we explicitly state that you can allow other identity providers for an organization such as Google, Github, ... via login policy?
- Should we state which policies we currently support (Password Complexity, Login Policy, IAM Access Preferences) -- instead of the last bullet?